src/main/scala/ch/openolitor/stammdaten/StammdatenRoutes.scala

```diff
@@ -254,6 +254,11 @@ trait StammdatenRoutes extends HttpService with ActorReferences
           delete(remove(kundentypId))
       }
 
+  private def vertriebeRoute(implicit subject: Subject, filter: Option[FilterExpr]): Route =
```

Comment:
This returns all Vertriebe, also the ones of the Zusatzabos. On the UI, it does not make sence to have all of them on the filter. I would offer three endpoints:
"vertriebe" - returning all, as now.
"hauptvertriebe" - returning only the ones from the Hauptabos.
"zusatzvertriebe" - returning ony the ones from the Zusatzabos.
(And then use the corresponding one on the Abos and Zusatzabos views.

Reply:
zusatzabos don't have vertriebe by its own. They are always associated to a mainabo with a vertrieb.

Reply:
True! Thx!